hack/generate-seed-crds.sh

```diff
@@ -33,6 +34,7 @@ fi
 
 output_dir="$(pwd)"
 file_name_prefix="$1"
+label=0
```

Comment:
Use

`add_deletion_protection_label={true,false}` to make it more expressive?

Reply:
PTAL c70578a59e8c5580f42c38b69a6582199d3c8415